Segfault using --vsp.sync

dcrwallet linux-amd64-v1.6.0-rc3 segfaults when attempting to use the `--vsp.sync` flag:
```
$ dcrwallet --promptpass --vsp.sync
2020-11-16 10:08:06.632 [INF] DCRW: Version 1.6.0-rc3+release (Go version go1.15.5 linux/amd64)
2020-11-16 10:08:07.898 [INF] WLLT: Opened wallet
Enter private passphrase:
2020-11-16 10:08:18.199 [INF] WLLT: The wallet has been unlocked without a time limit
2020-11-16 10:08:18.199 [INF] DCRW: Starting auto transaction creator
2020-11-16 10:08:18.200 [INF] RPCS: Listening on 127.0.0.1:9110
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0xb78e37]

goroutine 26 [running]:
decred.org/dcrwallet/internal/vsp.(*VSP).Sync(0x0, 0x10c6060, 0xc000304900)
        decred.org/dcrwallet@v1.6.0-rc3/internal/vsp/vsp.go:280 +0x37
created by main.run.func14
        decred.org/dcrwallet@v1.6.0-rc3/dcrwallet.go:419 +0xba
```

Seems like this feature would be useful given that vspd can require several fee payment attempts to pick up a new ticket. I'm assuming that's the intention of this flag, as afaik there is no other way to check if my tickets are on the vsp.

```
dcrwallet --help
Usage:
...
VSP Options:
      --vsp.url=                               Base URL of the VSP server
      --vsp.pubkey=                            VSP server pubkey
      --vsp.sync                               sync tickets to vsp
...
```

--- a/dcrwallet.go
+++ b/dcrwallet.go
@@ -246,7 +246,7 @@ func run(ctx context.Context) error {
 			passphrase = startPromptPass(ctx, w)
 		}
 
-		if cfg.EnableTicketBuyer && cfg.VSPOpts.URL != "" {
+		if cfg.VSPOpts.URL != "" {
 			changeAccountName := cfg.ChangeAccount
 			if changeAccountName == "" && cfg.CSPPServer == "" {
 				log.Warnf("Change account not set, using "+
@@ -420,7 +420,7 @@ func run(ctx context.Context) error {
 		}
 
 		loader.RunAfterLoad(func(w *wallet.Wallet) {
-			if cfg.VSPOpts.Sync {
+			if vspClient != nil && cfg.VSPOpts.Sync {
 				vspClient.ProcessManagedTickets(ctx, vspClient.Policy)
 			}
 

